button/button.css

```diff
@@ -0,0 +1,57 @@
+.btn {
+  font-size: 1rem;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+  outline: none;
+  color: #444444;
+}
+
+.btn-primary {
+  background-color: var(--theme-color);
+  color: white;
+}
+
+.btn.info {
+  background-color: rgb(124, 152, 212);
+  color: rgb(11, 47, 124);
+}
+
+.btn.success {
+  background-color: rgb(90, 224, 168);
+  color: rgb(18, 92, 66);
+}
+
+.btn.warning {
+  background-color: rgb(235, 200, 130);
+  color: rgb(255, 123, 0);
+}
+
+.btn.danger {
+  background-color: rgb(206, 98, 98);
+  color: rgb(119, 25, 25);
```

Comment:
colors can go to variables.

Reply:
Yaa.. we could do that.. but i thought to keep component css separate from my base css.
So that user will not have to load some extra css.